MicrosoftTeams.schema.json

```diff
@@ -126,6 +137,30 @@
             "description": "A color to use in conjunction with the icon. The value must be a valid HTML color code starting with '#', for example `#4464ee`.",
             "pattern": "^#[0-9a-fA-F]{6}$"
         },
+        "mobilePackageUrl": {
+            // this should be a private field
```

Comment:
this should not be exposed to public. where should we define this field. this is written here for discussion on this field.

Reply:
Regardless of whether we expose this in our schema or not we need to figure out how to prevent users from sideloading custom mobile module-based tabs. Maybe we can just limit this functionality to the Microsoft tenant or just inner rings (e.g. R0, R1, R2, R3.9)?

Reply:
As per the meeting, remove this section as well since it shouldn't be public